exercises/pig-latin/pig-latin.spec.js

```diff
@@ -1,7 +1,7 @@
 import PigLatin from './pig-latin';
```

Comment:
I have no problem going from a class-based solution to an object-based solution, but if we're going to do that, we can dispense with the `PigLatin` reference altogether:

```suggestion
import translator from './pig-latin';

  describe('Pig Latin', () => {
    
    test('translates a word beginning with a', () => {
```